deploy/rp-production-parameters.json

```diff
@@ -44,9 +44,6 @@
         "mdmFrontendUrl": {
             "value": ""
         },
-        "mdsdConfigVersion": {
```

Comment:
this change requires the RP-Config to be changed at the same time - prepare a PR for this.

Reply:
https://msazure.visualstudio.com/AzureRedHatOpenShift/_git/RP-Config/pullrequest/4405588